src/autoyast-rnc/partitioning.rnc

```diff
@@ -89,99 +95,109 @@ y2_partition =
+      opt_reserved_blocks?
+    )
   }
-opt_block_size = element opt_block_size { option_str, option_value }
-opt_blocksize = element opt_blocksize { option_str, option_value }
+
+option = MAP, (option_str, option_value )
+blank_option = MAP, (option_blank, option_str, option_value )
+opt_block_size = element opt_block_size { option }
+opt_blocksize = element opt_blocksize { option }
 opt_bytes_per_inode =
-  element opt_bytes_per_inode { option_str, option_value }
+  element opt_bytes_per_inode { option }
 opt_format =
-  element opt_format { option_blank, option_str, option_value }
-opt_hash = element opt_hash { option_blank, option_str, option_value }
-opt_inode_align = element opt_inode_align { option_str, option_value }
+  element opt_format { blank_option }
+opt_hash = element opt_hash { blank_option }
+opt_inode_align = element opt_inode_align { option }
 opt_max_inode_space =
-  element opt_max_inode_space { option_str, option_value }
-opt_raid = element opt_raid { option_str, option_value }
+  element opt_max_inode_space { option }
+opt_raid = element opt_raid { option }
 opt_reserved_blocks =
-  element opt_reserved_blocks { option_str, option_value }
-raid_name = element raid_name { text }
+  element opt_reserved_blocks { option }
+raid_name = element raid_name { STRING }
 
-raid_options = element raid_options { persistent_superblock? & chunk_size? & parity_algorithm? & raid_type? & device_order? & raid_name? }
+raid_options = element raid_options { MAP, (persistent_superblock? & chunk_size? & parity_algorithm? & raid_type? & device_order? & raid_name?) }
 persistent_superblock = element persistent_superblock { BOOLEAN }
-raid_type = element raid_type { text }
-chunk_size = element chunk_size { text }
-parity_algorithm = element parity_algorithm { text }
+raid_type = element raid_type { STRING }
+chunk_size = element chunk_size { STRING }
+parity_algorithm = element parity_algorithm { STRING }
 device_order = element device_order {
     LIST,
-    element device { text }*
+    element device { STRING }*
 }
 
-cache_mode = element cache_mode { "writethrough" | "writeback" | "writearound" | "none" }
+cache_mode = element cache_mode { STRING_ATTR, ("writethrough" | "writeback" | "writearound" | "none") }
 bcache_options = element bcache_options { cache_mode? }
 
-btrfs_name = element btrfs_name { text }
+btrfs_name = element btrfs_name { STRING }
 
-btrfs_options = element btrfs_options { data_raid_level? & metadata_raid_level? }
-data_raid_level = element data_raid_level { text }
-metadata_raid_level = element metadata_raid_level { text }
+btrfs_options = element btrfs_options { MAP, (data_raid_level? & metadata_raid_level?) }
+data_raid_level = element data_raid_level { STRING }
+metadata_raid_level = element metadata_raid_level { STRING }
 
 subvolumes = element subvolumes {
     LIST,
     subvolume*
 }
 
 # by default, AutoYaST exports list entries as 'listentry'
+# TODO: it can be configured to export it as subvolume, maybe do it?
```

Comment:
What?

Reply:
it is comment for that default export as listentry...We found on IRC that it is already fixed and we export it as subvolume already. I will remove it

Reply:
Ok, remove please